easybuild/easyconfigs/s/Shasta/Shasta-0.8.0-foss-2020b-fix_links.patch

```diff
@@ -0,0 +1,109 @@
+diff -ruN shasta-0.8.0_orig/dynamicExecutable/CMakeLists.txt shasta-0.8.0/dynamicExecutable/CMakeLists.txt
```

Comment:
@deniskristak Please add a short description on top of this patch, and mention the author (and then fix the SHA256 checksum in the Shasta easyconfig file)

Reply:
done